src/process/tags.md

```diff
@@ -0,0 +1,61 @@
+# Process tags
+
+Description of tags used for projects.
+
+## Status
+- refine `#c377e0` - issue already have resolving code, but code needs some fixes.
```

Comment:
These statuses is applicable only for issues or for PR too?

Reply:
`refine`, `take it` and `discuss` are not statuses. Status answers an question like: "What is its condition?"
So better to use:
`refine -> in improving` (refine does not fit, because it is mean - remove impurities or unwanted elements from (a substance), typically as part of an industrial process. better to use improve)
`take it -> ready for work`
`discuss -> in discussion`

Reply:
@artemiy312 ,

> refine, take it and discuss are not statuses

i think, we are not needed to name groups purely. So, you can propose your own name for this group or regroup all tags in the whole

> refine -> in improving

"remove impurities or unwanted elements from (a substance), typically as part of an industrial process" - it's pretty what we do with code on review fixes, is not? :)

improve is too common word in my test. `refine` sounds like final cosmetics. So, it's always about code. But `improve` is not. It can be setted instead of `needs info`, for example.

> take it -> ready for work, discuss -> in discussion

emm, my variants have much less symbols with pretty the same essential. So, why we are needed in additional symbols?